ts/components/services/NewServiceListItem.tsx

```diff
@@ -65,35 +92,37 @@ export default class NewServiceListItem extends React.PureComponent<
       `service-switch`
     );
   };
+
+  private onItemSwitchValueChanged(value: boolean) {
```

Comment:
This is the same method rewritten with some improvements and using if statement instead of ternary operator (I guess it is more readable because there are few lines of code)
```typescript
private onItemSwitchValueChanged(value: boolean) {
    const potService = this.props.item;
    if (this.props.onItemSwitchValueChanged !== undefined) {
      const onItemSwitchValueChanged = this.props.onItemSwitchValueChanged;
      return pot.toUndefined(
        pot.map(potService, service => {
          if (
            !pot.isUpdating(this.props.profile) &&
            this.state.switchValue !== value
          ) {
            this.setState({
              switchValue: value
            });
            onItemSwitchValueChanged(service, value);
          }
        })
      );
    }
    return undefined;
  }
```